release.go

```diff
@@ -273,22 +292,16 @@ func buildReleaseCommits(ctx context.Context, ghClient *githubClient, commits []
 		}
 
 		if gen.UsePullRequestMetadata {
-			prNumber, ok := commit.PullRequestNumber()
-			if !ok {
-				continue
-			}
-			c.PullRequestNumber = prNumber
-
-			var err error
-			pr, ok := prs[prNumber]
-			if !ok {
-				pr, err = ghClient.getPullRequest(ctx, event.Owner, event.Repo, prNumber)
-			}
+			pr, err := getPullRequest(commit)
 			if err != nil {
-				return nil, err
+				// only error logging, ignore error
+				log.Printf("Failed to get pull request: %v\n", err)
```

Comment:
The PR Number retrieved from the commit subject is now the user's input value, so only logging is used when an error occurs. Commits that fall into this case are assumed to be output in the conventional form without PR metadata.